prompt.go

```diff
@@ -1,16 +1,20 @@
 package prompt
 
-import "github.com/howeyc/gopass"
-import "strings"
-import "strconv"
-import "fmt"
+import (
```

Comment:
I would have preferred to keep one per line as per the original code but well...